transcom/mymove
# Int b 19303 - Added "SIT Reimbursement" to "Incentive Factors" on PPM Closeout screen

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19303)

## Summary

This branch adds a "SIT Reimbursement" item to the "Incentive Factors" section of the PPM Closeout review screen.

The SIT Reimbursement amount, if accepted, is now added to the GCC, Gross Incentive, Remaining Incentive, and Authorized Receipt Total amounts.

### How to test

1. Create a PPM move as a customer and sign
2. Approve move as an SC
3. Upload PPM docs as a customer, be sure to include at least one "Storage" receipt under "Expenses". Sign.
4. Pull up the move in PPM Closeout Queue as an SC, click "Review Documents"
5. If you accept the Storage receipt document, you should see that total reflected in all of the fields listed above. If you go back at reject it, these fields should decrease by the Storage receipt's dollar amount.

![Screenshot 2024-05-31 at 1 39 39 PM](https://github.com/transcom/mymove/assets/146959895/a0105f8c-a8b6-4700-80e7-65ea722c505a)
![Screenshot 2024-05-31 at 1 39 49 PM](https://github.com/transcom/mymove/assets/146959895/6f1df316-dcde-4e42-b837-a399f9bf5080)
